src/cider/common/hashtable/HashTable.h

```diff
@@ -1077,15 +1071,11 @@ class HashTable : private boost::noncopyable,
     return const_cast<std::decay_t<decltype(*this)>*>(this)->find(x, hash_value);
   }
 
-  ALWAYS_INLINE bool erase(const Key& x)
   // requires Grower::performs_linear_probing_with_single_step
-  {
-    return erase(x, hash(x));
-  }
+  ALWAYS_INLINE bool erase(const Key& x) { return erase(x, hash(x)); }

```

Comment:
where defined `ALWAYS_INLINE`

Reply:
It seems to put `ALWAYS_INLINE` behind return type info in velox and ck.

Reply:
`ALWAYS_INLINE` is defined in `funcannotations.h`, files in common folder will include this header file?

Reply:
I got it, I think it should be included but it's strange that it compiled well now..